wrapanapi/ec2.py

```diff
@@ -237,6 +251,15 @@ def stack_exist(self, stack_name):
         else:
             return bool(stacks)
 
+    def list_stacks(self, stack_status_filter=stack_states['active']):
```

Comment:
Although it's counter-intuitive, to maintain API consistency with the other `list_<something>` functions, let's call this `list_stack`.